.github/scripts/setup_runners.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+set -e
+
+#####################################
+# Variables / Parameters / Settings #
+#####################################
+
+# This script allows users to manually assign parameters
+if [ "$#" -ne 3 ] || [ "$1" = "--help" ] || [ "$1" = "-h" ]
+then
+  echo "Please assign at 3 parameters when running this script"
+  echo "Example: $0 \$ACTION \$EC2_INSTANCE_NAMES(,) \$GITHUB_TOKEN"
+  echo "Example: $0 \"run\" \"odfe-rpm-ism,odfe-rpm-sql\" \"<GitHub PAT>\""
+  exit 1
+fi
```

Comment:
It would be super helpful if the help message included a brief description of what this script does, and what the parameters mean. The examples are useful, but a formal description of parameters (including a list of valid $ACTIONs) would be even better.

Reply:
You are right. As of now these scripts are only used internally, but I will remember that and add some comments in later attempts.